applications/FluidDynamicsApplication/python_scripts/navier_stokes_compressible_explicit_solver.py

```diff
@@ -158,3 +162,34 @@ def _CreateEstimateDtUtility(self):
                 self.settings["time_stepping"])
 
         return estimate_dt_utility
+
+    def _CheckDeprecatedSettings(self, custom_settings):
+        if custom_settings.Has("shock_capturing"):
+            KratosMultiphysics.Logger.PrintInfo(
+                '::[NavierStokesCompressibleExplicitSolver]:: "',
+                "\n".join(['Setting "shock_capturing" is deprecated. Please use:',
+                           '   {',
+                           '       "shock_capturing_settings" : {',
+                           '            "type" : "physics_based"',
+                           '       }',
+                           '   }',
+                           'to maintain the same functionality with the newer syntax.'
+                ]))
+
+            custom_settings.RemoveValue("shock_capturing")
+            # Not adding new syntax -> Using defauts
```

Comment:
Considering that this is a feature under development since not long ago I wouldn't get into the backwards compatibility complexities.

Reply:
Okay